fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4563,8 +4432,10 @@ class FMCMainDisplay extends BaseAirliners {
      * Check if string is in place/bearing/distance format
      * @param {String} s
      * @returns true if pbd
+     * @deprecated use {@link Fmgc.WaypointEntryUtils.isPbdFormat}
      */
     isPbdFormat(s) {
+        return Fmgc.WaypointEntryUtils.isPbdFormat(s);
         const pbd = s.match(/^([^\/]+)\/([0-9]{1,3})\/([0-9]{1,3}(\.[0-9])?)$/);
         return pbd !== null && this.isPlaceFormat(pbd[1]);
```

Comment:
Dead Code, again check that regex is correctly implemented in WayPointEntryUtils